scriptworker/task.py

```diff
@@ -229,16 +230,15 @@ def guess_content_type(path):
     }
     args = [get_task_id(context.claim_task), get_run_id(context.claim_task),
             target_path, payload]
+    _compress_if_text_plain(path, payload['contentType'])
```

Comment:
I'm not 100% sure we should compress all artifacts if `text/plain`.  If this ends up compressing `chainOfTrust.json.asc`, do we automatically decompress on download or will we have to decompress before reading?

If we're not sure, we might want to do the compression when we copy the logs to the `upload_dir`.